.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -14,6 +14,27 @@ body:
       options:
       - label: I have searched the existing issues
         required: true
+  - type: dropdown
+    id: deploy-type
+    attributes:
+      label: Deploy type
+      description: |
+        How did you deploy the Dashboard?
+
+        Eg. what is the repo location, `odh-manifests` or `odh-dashboard`?
+      multiple: false
+      options:
+        - Using an OpenDataHub main version (eg. `v1.6.0`)
```

Comment:
mmmm this is great, it will allow us to discriminate some issues.

Reply:
Yeah, might need to improve this over time -- but at least it has some information.

Probably need some details how to find this information maybe 🤔